Rules/CommonScripts/DefaultVotes.as

```diff
@@ -455,10 +463,24 @@ void onMainMenuCreated(CRules@ this, CContextMenu@ menu)
 
 	bool can_skip_wait = getSecurity().checkAccess_Feature(me, "skip_votewait");
 
+	bool duplicatePlayer = me.getUsername().find("~") > -1;
```

Comment:
replace with ``bool duplicatePlayer = VoteCheckFunctor::PlayerCanVote(me);``

Reply:
Can't do that. That line isn't within a class that inherits from `VoteCheckFunctor`.
![image](https://user-images.githubusercontent.com/30195802/118402818-f66f8300-b6ae-11eb-8b56-7807094d6729.png)

Reply:
Ah that makes sense :+1: